libs/langchain/langchain/utilities/serpapi.py

```diff
@@ -77,29 +60,31 @@ def validate_environment(cls, values: Dict) -> Dict:
         return values
 
     async def arun(self, query: str, **kwargs: Any) -> str:
-        """Run query through SerpAPI and parse result async."""
+        """Run query through SerpApi and parse result async."""
         return self._process_response(await self.aresults(query))
 
     def run(self, query: str, **kwargs: Any) -> str:
-        """Run query through SerpAPI and parse result."""
+        """Run query through SerpApi and parse result."""
         return self._process_response(self.results(query))
 
     def results(self, query: str) -> dict:
-        """Run query through SerpAPI and return the raw result."""
+        """Run query through SerpApi and return the raw result."""
         params = self.get_params(query)
-        with HiddenPrints():
-            search = self.search_engine(params)
-            res = search.get_dict()
```

Comment:
The bug with unneeded printing was fixed in https://github.com/serpapi/google-search-results-python/pull/31 and released in 2.4.2 (that version is unsed Langchain).